package.json

```diff
@@ -15,5 +21,25 @@
   "dependencies": {
     "bluebird": "^2.9.26",
     "lodash": "^3.9.2"
+  },
+  "devDependencies": {
+    "babel-core": "^5.8.20",
+    "babel-loader": "^5.3.2",
+    "karma": "^0.13.3",
+    "karma-coverage": "^0.4.2",
+    "karma-mocha": "^0.2.0",
+    "karma-phantomjs-launcher": "^0.2.0",
+    "karma-sinon-chai": "^1.0.0",
+    "karma-spec-reporter": "0.0.20",
+    "karma-threshold-reporter": "^0.1.15",
+    "karma-webpack": "^1.7.0",
+    "mocha": "^2.2.5",
+    "phantomjs": "^1.9.17",
+    "webpack": "^1.10.5"
+  },
+  "scripts": {
+    "install": "npm run build",
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "test": "./node_modules/karma/bin/karma start ./karma.config.js"
```

Comment:
Interesting, thought I tried that and had an issue, but will double check. That would definitely be a lot cleaner.